docs/_sass/application.scss

```diff
@@ -82,3 +83,21 @@ $govuk-assets-path: "/govuk-google-analytics/assets/";
   color: govuk-colour("dark-grey");
   @include govuk-font(19);
 }
+
+.highlight-json {
+  padding: govuk-spacing(2);
+  border: solid 1px $govuk-border-colour;
+  background: govuk-colour("light-grey");
+}
+
+.highlight-json__brace {
```

Comment:
Empty class!

Reply:
Thanks, have removed (and references to it).